ups/eupspkg.cfg.sh

```diff
@@ -67,4 +67,14 @@ install()
 	default_install
 
 	cp -r include "$PREFIX/"
+	if [[ $OSTYPE == darwin* ]]; then
+		curdir=`pwd`
+		cd $PREFIX/lib
+		galsim_name=`ls libgalsim.*.dylib`
```

Comment:
This is potentially fragile if in the future there happens to be more than one version of the library at once. galsim_name would then be nonsense

Reply:
How would you do this otherwise?  I don't want to have to change this code if the galsim version changes.

Reply:
`galsim_name_array=( ( ls libgalsim.*.dylib ) )
galsim_name=${galsim_name_array[${#galsim_name_array[@]}-1]}`

This will list the directory, save it as an array, and then select the last element of the array (the length minus 1) This presumes the structure of how the libs are named will not change such that ls will always put then in ascending order, which is probably a reasonable assumption.

This may be something that does not need worried about, I am just trying to guard against an unexpected failure where someone (maybe the build system) does something weird in the future

Reply:
I'd be inclined to leave it as the simple version since we are in control of when galsim is updated in this package.

Reply:
I ended up grabbing the version from the python version file.  I think that is the single source of truth for this package.